packages/terra-worklist-data-grid/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "focus-trap-react": "^6.0.0",
```

Comment:
Is the newer focus library that Steven discovered?

Reply:
No, this is the existing library that is used throughout the rest of Terra.  I kept the version consistent with the rest of terra-framework.